frontend/admin/src/js/components/classification/ClassificationTable.tsx

```diff
@@ -87,7 +87,7 @@ export const ClassificationTable: React.FC<
           tableEditButtonAccessor(
             d.id,
             editUrlRoot,
-            `${d.name?.[locale]} ${d.group}-${d.level}`,
+            `${d.name?.[locale]} ${d.group}-0${d.level}`,
```

Comment:
Was this intentional?

Reply:
> Was this intentional?

Yes, it was missed in another [PR](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4826) and slipped into this one.

Reply:
Thanks Matt